pkg/trimaran/loadvariationriskbalancing/loadvariationriskbalancing.go

```diff
@@ -111,14 +111,14 @@ func (pl *LoadVariationRiskBalancing) Score(ctx context.Context, cycleState *fra
 	if cpuOK {
 		cpuScore = cpuStats.computeScore(pl.collector.args.SafeVarianceMargin, pl.collector.args.SafeVarianceSensitivity)
 	}
-	klog.V(6).Infof("pod:%s; node:%s; CPUScore=%f", pod.GetName(), nodeName, cpuScore)
+	klog.V(6).InfoS("pod:Name; node:nodeName; CPUScore=", "Name", klog.KObj(pod), "nodeName", nodeName, "cpuScore", cpuScore)
```

Comment:
Ditto. The first field should be a generic msg:

```suggestion
	klog.V(6).InfoS("CPUScore for pod", "pod", klog.KObj(pod), "nodeName", nodeName, "cpuScore", cpuScore)
```

BTW: I think you also missed migrating L94. Could you do a thorough check?